### DISCUSSION TITLE
Using MIKE IO 1D in Colab?

### DISCUSSION THREAD

**Discussion by alessio-pugliese** [DHI/mikeio1d#47] [Q&A] [answered]
I tried running an instance of mikeio1d in Google Colab. While mikeio works in Colab as documented in the readme page, mikeio1d seems to produce errors when retrieving libraries. Basically it can be installed but can't import Res1D. Is there a reason for this or could be a bug? I can show some output if needed. 
Thanks,
Alessio

**Comment by ecomodeller** [comment, marked as answer]
Google Colab runs on Linux. MIKE IO 1D is not yet supported on Linux.

This was actually the main reason for separating MIKE IO 1D from MIKE IO into a separate project.

**Reply by alessio-pugliese** [reply]
Oh I see. Thanks for your reply. Hopefully there might be some interest from the community to a porting in linux. For the moment I'll try it out on my local python in windows. 
Cheers
